jenkins/shared-library/vars/publish.groovy

```diff
@@ -57,7 +66,7 @@ void pythonPublisher(Map config, String phase) {
 	}
 
 	credentialsId = PYTHON_CREDENTIALS_ID
-	if (phase == 'alpha') {
+	if (isAlphaRelease(phase)) {
 		credentialsId = TEST_PYTHON_CREDENTIALS_ID
 	}
```

Comment:
any reason not to replace pypt script too?

Reply:
Nope, can remove the pypi script also